### PR TITLE
feat: added toast to display event edit status

### LINKED ISSUE #1274: after edit some event details and press "save" - better to add notification about operation status
### Discuss your ideas or share your views: 
after edit some event details and press "save" better to add notification about operation status - if the changes saved successfully or some error occurred

### PR DESCRIPTION
**Pull request by awais-codes**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #1274

**What I did:**
Added a toast message on successful and failed edit of an Event

### CODE REVIEW

File: client/src/modules/dashboard/Events/pages/EditEventPage.tsx [hunk 1]
```diff
@@ -67,9 +70,17 @@ export const EditEventPage: NextPage = () => {
       });
 
       if (event.data) {
+        toast({
+          title: `Event "${event.data.updateEvent.name}" updated successfuly!`,
+          status: 'success',
+        });
         await router.push('/dashboard/events');
```
Comment (gikf) [inline client/src/modules/dashboard/Events/pages/EditEventPage.tsx:77]:
```suggestion
        await router.push('/dashboard/events');
        toast({
          title: `Event "${event.data.updateEvent.name}" updated successfuly!`,
          status: 'success',
        });
```
What do you think about swapping order here? Right now the appearing animation happens twice, after clicking save button when data is saved and after opening `/dashboard/events`. With swapped order toast appears just in the dashboard.

Comment (Awais753) [reply client/src/modules/dashboard/Events/pages/EditEventPage.tsx]: I could not notice this on my end :/. Anyway, I made the suggested change.

**Review by gikf — APPROVED** [review]
Looks good!

**Comment by gikf** [comment]
Hi @Awais753. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

@all-contributors would you be so kind and add @Awais753 for code?

**Pull request merged**